_install-and-configure/install-opensearch/index.md

```diff
@@ -39,9 +39,10 @@ The OpenSearch distribution for Linux ships with a compatible [Adoptium JDK](htt
 
 OpenSearch Version | Compatible Java Versions | Bundled Java Version
 :---------- | :-------- | :-----------
-1.0 - 1.2.x | 11, 15    | 15.0.1+9
-1.3.x       | 8, 11, 14 | 11.0.14.1+1
-2.0.0       | 11, 17    | 17.0.2+8
+1.0 - 1.2.x    | 11, 15     | 15.0.1+9
+1.3.x          | 8, 11, 14  | 11.0.22+7
+2.0.0 - 2.11.x | 11, 17     | 17.0.2+8
```

Comment:
```suggestion
2.0.0 -- 2.11.x  | 11, 17     | 17.0.2+8
```

Reply:
@Naarcha-AWS apologies, why we need `--`? We use `1.0 - 1.2.x` few lines above ...